examples/plot_interpolate_bad_channels.py

```diff
@@ -51,7 +49,7 @@
 evoked_before.plot(exclude=[])
 
 # compute interpolation (also works with Raw and Epochs objects)
-evoked_after = evoked_before.copy().interpolate_bads_eeg()
+evoked_after = evoked_before.interpolate_bads()
 
-# plot interpolated (prevsious bads)
+# plot interpolated (previous bads)
 evoked_after.plot(exclude=[])
```

Comment:
Awesome. This reminds me. The bads are plotted in a different call from the non-bads, right? We should set the bad plot call to have `zorder=-1`. Can you do it here?